great_expectations/core/expectation_suite.py

```diff
@@ -598,19 +599,61 @@ def send_usage_event(self, success: bool):
                 success=success,
             )
 
+    def add_expectation_configurations(
+        self,
+        expectation_configurations: List[ExpectationConfiguration],
+        send_usage_event: bool = True,
+        match_type: str = "domain",
+        overwrite_existing: bool = True,
+    ) -> List[ExpectationConfiguration]:
+        """
+        Args:
+            expectation_configurations: The List of candidate new/modifed "ExpectationConfiguration" objects for Suite.
+            send_usage_event: Whether to send a usage_statistics event. When called through ExpectationSuite class'
+                public add_expectation() method, this is set to `True`.
+            match_type: The criteria used to determine whether the Suite already has an "ExpectationConfiguration"
+                object, matching the specified criteria, and thus whether we should add or replace (i.e., "upsert").
+            overwrite_existing: If "ExpectationConfiguration" already exists, this will cause it to be overwritten if
+                True and raise an error if False.
+
+        Returns:
+            The List of "ExpectationConfiguration" objects attempted to be added or replaced (can differ from the list
+            of "ExpectationConfiguration" objects in "self.expectations" at the completion of this method's execution).
+        Raises:
+            More than one match
+            One match if overwrite_existing = False
```

Comment:
Is this section accurate? "Raises" should outline different errors/exceptions and why they may be raised.

Reply:
@cdkini I believe that since `add_expectation()` is called in a loop, the same exceptions propagate upwards.  Thanks.

Reply:
I'd write something like:
```
ValueError or DataContextError under certain situations; please refer to `add_expectation` for more information.
```